feat: refactoring renaming planters to growers

tranquanghuy0801 opened the pull request:
resolve the issue #148

nmcharlton commented on src/context/GrowerContext.js:
```diff
@@ -0,0 +1,125 @@
+import React, { useState, createContext } from 'react';
```
git has picked up this renaming as a deletion and creation of a new file.
Are you able to redo this file using a `git mv` command so we keep the history and can compare?

nmcharlton commented on src/common/policy.js:
```diff
@@ -17,12 +17,12 @@ const policy = {
     {
-      name: 'list_planter',
-      description: 'Can view planters',
+      name: 'list_grower',
```
Unfortunately, this policy name value is encoded in records in the `admin_role` table in the database.
We should aim to eventually change it, but it will require some coordination across the deployments.

nmcharlton commented on src/models/auth.js:
```diff
@@ -10,7 +10,7 @@ const POLICIES = {
   LIST_TREE: 'list_tree',
   APPROVE_TREE: 'approve_tree',
-  LIST_PLANTER: 'list_planter',
+  LIST_GROWER: 'list_GROWER',
```
The key could change to `LIST_GROWER` but the value will need to remain as `list_planter` for compatibility with the production database.

tranquanghuy0801 commented:
@nmcharlton , I have changed the filename of all relevant to the original ones as you requested.

nmcharlton commented:
> @nmcharlton , I have changed the filename of all relevant to the original ones as you requested.

Sorry Harry, that's not what I meant. There was one file where we'd lost the history during the renaming, but all the other files were fine.

tranquanghuy0801 commented:
> > @nmcharlton , I have changed the filename of all relevant to the original ones as you requested.
> 
> Sorry Harry, that's not what I meant. There was one file where we'd lost the history during the renaming, but all the other files were fine.

Sorry Nick, I have changed back to what I did initially. The only two files I keep the original names are src/components/PlanterContext.js and cypress/components/planter.spec.py.js as git recognized these files as being new.
